notes/D02-classification_current_cogScores_fromtaxa_bygroup.jl

```diff
@@ -134,15 +135,12 @@ prediction_df = @chain cogscore_taxa_df begin
     subset(:ageMonths => x -> x .<= max_prediction_ageMonths)
 end
 
-RandomForestClassifier= @load RandomForestClassifier pkg=DecisionTree
-classification_00to06_results = train_randomforest_current_classifier(prediction_df; n_trials = 5, min_age = 0.0, max_age=6.0, split_proportion=0.75, train_rng=ml_rng)
-classification_06to12_results = train_randomforest_current_classifier(prediction_df; n_trials = 5, min_age = 6.0, max_age=12.0, split_proportion=0.75, train_rng=ml_rng)
-classification_12to18_results = train_randomforest_current_classifier(prediction_df; n_trials = 5, min_age = 12.0, max_age=18.0, split_proportion=0.75, train_rng=ml_rng)
-classification_18to24_results = train_randomforest_current_classifier(prediction_df; n_trials = 5, min_age = 18.0, max_age=24.0, split_proportion=0.75, train_rng=ml_rng)
-
-using JLD2
-
-JLD2.@save "models/results_classification_currentCogScores_00to06_onlytaxa.jld" classification_00to06_results
-JLD2.@save "models/results_classification_currentCogScores_06to12_onlytaxa.jld" classification_06to12_results
-JLD2.@save "models/results_classification_currentCogScores_12to18_onlytaxa.jld" classification_12to18_results
-JLD2.@save "models/results_classification_currentCogScores_18to24_onlytaxa.jld" classification_18to24_results
\ No newline at end of file
+RandomForestClassifier= MLJ.@load RandomForestClassifier pkg=DecisionTree
+classification_currentCogScores_00to06_fromtaxa_results = train_randomforest_current_classifier(prediction_df; n_trials = 5, min_age = 0.0, max_age=6.0, split_proportion=0.75, train_rng=ml_rng)
+JLD2.@save "models/classification_currentCogScores_00to06_fromtaxa_results.jld" classification_currentCogScores_00to06_fromtaxa_results
```

Comment:
A general style note: I don't love the super long variable / function names, especially when thy're only being used on 2 lines. I'd prefer something like the following:

```suggestion
let
    n_trials = 5
    min_age = 0.0
    max_age = 6.0
    split_proportion = 0.75
    train_rng = ml_rng
    path = "models/classification_currentCogScores_00to06_fromtaxa_results.jld"
    
    mod = train_randomforest_current_classifier(prediction_df; n_trials, min_age , max_age, split_proportion, train_rng)
    JLD2.@save path mod
end
```

Then repeat the block and just change the values at the top. This also makes it easier to diff, and to modify things later. I don't feel strongly enough about it to make the change now, but going forward, please try to keep this in mind.

You might also consider using dispatch / types instead of super long function names. Eg instead of `train_randomforest_current_classifier`, it could be

```
struct Current end
struct Future end
struct Classifier end
struct Regression end

function trainrf(::Current, ::Classifier, args...)
    # etc...
end

function trainrf(::Future, ::Regression, args...)
    # etc...
end
```